lib/helper.py

```diff
@@ -182,3 +182,47 @@ def remove_file(src, dest):
         if os.path.exists(d_file):
             os.remove(d_file)
         logger.debug("%s file deleted" % d_file)
+
+
+class PipMagager:
+    def __init__(self, base_fw=[], opt_fw=[], kvm_fw=[], enable_kvm=False):
+        """
+        helper class to parse, install, uninstall pip package from user config
+        """
+        self.uninstallitems = base_fw + opt_fw + kvm_fw
+        if enable_kvm:
+            self.installitems = self.uninstallitems
+        else:
+            self.installitems = base_fw + opt_fw
+
+        self.install_packages = []
+        self.uninstall_packages = []
+        self.pip_cmd = "pip%s" % sys.version_info[0]
+        for item in self.uninstallitems:
+            self.uninstall_packages.append(item[0])
+        for item in self.installitems:
+            if item[1]:
+                if 'git' in item[1]:
```

Comment:
Can we have `item[1].startswith('git')` here?

Reply:
yep, makes sense, will change..